Makefile

```diff
@@ -1,15 +1,31 @@
+.PHONY: clean-pyc init lint test install create-db fetch-issues tweet-issues
 
-.PHONY: clean-pyc init lint test
 
 lint:
 	flake8 --exclude=.tox
 
-#init: 
-#	pip install -r requirements.txt 
 
-test:	
+install:
+	pip install -r requirements.txt
+
+
+create-db:
+	python first_timers/run.py --creds-path credentials.json --db-path data/db.json --only-save --create
+
+
+fetch-issues:
+	python first_timers/run.py --creds-path credentials.json --db-path data/db.json --only-save
+
+
+tweet-issues:
+	python first_timers/run.py --creds-path credentials.json --db-path data/db.json
```

Comment:
These 3 steps should not be part of Makefile. This needs to be done manually and separately as per the requirement during the setup. 

Can you remove these 3 steps.